nixos/modules/services/mail/isync.nix

```diff
@@ -0,0 +1,72 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.isync;
+in {
+
+  options.services.isync = {
+    enable = mkEnableOption "Isync, a software to dispose your mailbox(es) as a local Maildir(s)";
+
+    install = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Whether to install a user service for Isync. Once
+        the service is started, emails will be fetched automatically.
+
+        The service must be manually started for each user with
+        "systemctl --user start isync" or globally through
+        <varname>services.isync.enable</varname>.
+      '';
+    };
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.isync;
+      defaultText = "pkgs.isync";
+      description = "Isync derivation to use.";
+    };
+
+    path = mkOption {
+      type = types.listOf types.path;
+      default = [];
+      example = literalExample "[ pkgs.pass pkgs.bash pkgs.notmuch ]";
+      description = "List of derivations to put in Isync's path.";
+    };
+
+    onCalendar = mkOption {
+      type = types.str;
+      default = "*:0/3"; # every 3 minutes
+      description = "How often is isync started. Default is '*:0/3' meaning every 3 minutes. See systemd.time(7) for more information about the format.";
```

Comment:
There's `<literal>foo</literal>` for literal strings, and there's this incantation for man page references: `<citerefentry><refentrytitle>man</refentrytitle><manvolnum>1</manvolnum></citerefentry>`